after-effects/after-effects-quiz.md

```diff
@@ -74,10 +74,10 @@
 
 #### Q11. What properties are added to a 3D layer? (Select three items.)
 
-- [ ] Z Rotation
-- [ ] Anchor Point
+- [x] Z Rotation
+- [x] Anchor Point
 - [ ] Material Options
-- [ ] Position
+- [x] Position
```

Comment:
it is better to keep only one, can you leave your favorite, thanks

Reply:
@Ebazhanov can you please elaborate i didn't understand.

Reply:
sure, please leave only one answer with [x]

Reply:
@Ebazhanov actually, the question asked to select "three items", that is why I selected three options. Hope you understand this

Reply:
my bad did not read it properly, you are right!!!

Reply:
yeah it happens, by the way thanks, I am new to contributing and stuff. Got to learn a lot from all this